pkg/pillar/cmd/zedagent/localinfo.go

```diff
@@ -88,47 +95,38 @@ func localAppInfoPOSTTask(ctx *getconfigContext) {
 	}
 }
 
-func sendLocalAppInfo(ctx *getconfigContext) {
+func sendLocalAppInfo(ctx *getconfigContext) *profile.LocalAppCmdList {
```

Comment:
Would it make sense to rename this to postLocalAppInfo and add a comment that it sends the status and receives app commands?

Reply:
done